src/common/commands_parser.rs

```diff
@@ -224,6 +225,40 @@ impl ExplainArgs {
     }
 }
 
+/// The arguments used by `pcr` command
+pub struct PcrArgs {
+    /// Path to the file needed for hashing
+    pub path: String,
+    /// The type of file we need to hash
+    pub pcr_type: PcrType,
+}
+
+impl PcrArgs {
+    /// Construct a new `PcrArgs` instance from the given command-line arguments.
+    pub fn new_with(args: &ArgMatches) -> NitroCliResult<Self> {
+        let mut val_name = "input";
+        let mut pcr_type = PcrType::DefaultType;
+        if args.is_present("signing-certificate") {
+            val_name = "signing-certificate";
+            pcr_type = PcrType::SigningCertificate;
+        }
```

Comment:
No need for mutable variables

```suggestion
let (val_name, pcr_type) = match args.is_present("signing-certificate") {
    true => ("signing-certificate", PcrType::SigningCertificate),
    false => ("input", PcrType::DefaultType),
}
```